tests/Home.js

```diff
@@ -26,108 +24,131 @@ describe("Home page", () => {
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
     await (await HamburgerMenu.goodButton).click();
     await expect(await Menu.header).toHaveText("GOOD");
-    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
 
-    browser.execute(() => window.scrollTo(0, 500));
+    await HomePage.scrollDown();
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
     await (await HamburgerMenu.omgButton).click();
     await expect(await Menu.header).toHaveText("OMG");
   });
 
-  it("Hamburger menu last part", async () => {
+  it("Check the hamburger menu last part functionality", async () => {
+    await HomePage.scrollDown();
     await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
-    await browser.execute(() => window.scrollTo(0, 500));
     await (await HamburgerMenu.hamburgerMenuButton).click();
+    await (await HamburgerMenu.aboutButton).waitForDisplayed();
     await (await HamburgerMenu.aboutButton).click();
     await expect(await Menu.title).toHaveText("About Us");
 
-    await browser.execute(() => window.scrollTo(0, 500));
+    await HomePage.scrollDown();
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
+    await (await HamburgerMenu.privacyPolicyButton).waitForDisplayed();
     await (await HamburgerMenu.privacyPolicyButton).click();
     await expect(await Menu.title).toHaveText("Privacy Policy");
 
-    await browser.execute(() => window.scrollTo(0, 500));
+    await HomePage.scrollDown();
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
+    await (await HamburgerMenu.dmcaButton).waitForDisplayed();
     await (await HamburgerMenu.dmcaButton).click();
     await expect(await Menu.title).toHaveText("DMCA Policy");
 
-    await browser.execute(() => window.scrollTo(0, 500));
+    await HomePage.scrollDown();
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
+    await (await HamburgerMenu.advertisingButton).waitForDisplayed();
     await (await HamburgerMenu.advertisingButton).click();
     await expect(await Menu.title).toHaveText("Advertising Interest");
 
-    await browser.execute(() => window.scrollTo(0, 500));
+    await HomePage.scrollDown();
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
+    await (await HamburgerMenu.closeMenu).waitForDisplayed();
     await (await HamburgerMenu.closeMenu).click();
   });
 
-  it("Footer", async () => {
-    await (await HomePage.aboutFooterButton).waitForDisplayed();
+  it("Check the first part of footer functionality", async () => {
```

Comment:
You can say that this step is checking the functionality of the links from the footer element. When you say  "the first part" can be understood as the first link.